packages/react-components/src/components/utils/SendBoxUtils.ts

```diff
@@ -206,7 +206,7 @@ export const insertImagesToContentString = async (
 /**
  * @internal
  */
-export const removeBrokenImageContent = (content: string): string => {
+export const removeBrokenImageContentAndClearImageStyle = (content: string): string => {
```

Comment:
Should it be something like `...ClearImageSizeStyles` as we don't remove all styles, only width and height?